pkg/container/qemu_runner.go

```diff
@@ -393,6 +385,19 @@ func (bw *qemu) WorkspaceTar(ctx context.Context, cfg *Config) (io.ReadCloser, e
 	// We could just cp -a to /mnt as it is our shared workspace directory, but
 	// this will lose some file metadata like hardlinks, owners and so on.
 	// Example of package that won't work when using "cp -a" is glibc.
+	retrieveCommand := "cd /home/build && tar cvpf - --xattrs --acls --exclude='*fifo*' melange-out"
+	// we append also all the necessary files that we might need, for example Licenses
+	// for license checks
+	for _, v := range extraFiles {
+		retrieveCommand = retrieveCommand + " " + v
+	}
```

Comment:
I don't think this works if the license-path includes a subdirectory?

I am seeing a failure on: https://github.com/chainguard-dev/extra-packages/blob/main/camunda-zeebe-8.6.yaml

Reply:
This will fix it: https://github.com/chainguard-dev/melange/pull/1985
it's a problem with the unpack, not the packing